src/nanorc/logbook.py

```diff
@@ -43,89 +37,33 @@ def message_on_stop(self, message:str, apparatus:str):
-        self.current_run = None
-        self.current_run_type = None
-
-
-    def _send_message(self, subject:str, body:str, command:str):
-        elisa_arg = copy.deepcopy(self.elisa_arguments)
-        sso = {"ssocookie": credentials.generate_new_sso_cookie(self.website)}
-        elisa_arg.update(sso)
-
-        elisa_inst = Elisa(**elisa_arg)
-        try:
-            answer = None
-            if not self.current_id:
-                self.log.info("ELisA logbook: Creating a new message thread")
-                message = MessageInsert()
-                message.author = credentials.user
-                message.subject = subject
-                for attr_name, attr_data in self.message_attributes[command].items():
-                    if attr_data['set_on_new_thread']:
-                        setattr(message, attr_name, attr_data['value'])
-                message.systemsAffected = ["DAQ"]
-                message.body = body
-                answer = elisa_inst.insertMessage(message)
-                
-            else:
-                self.log.info(f"ELisA logbook: Answering to message ID{self.current_id}")
-                message = MessageReply(self.current_id)
-                message.author = credentials.user
-                message.systemsAffected = ["DAQ"]
-                for attr_name, attr_data in self.message_attributes[command].items():
-                    if attr_data['set_on_reply']:
-                        setattr(message, attr_name, attr_data['value'])
-                message.body = body
-                answer = elisa_inst.replyToMessage(message)
-            self.current_id = answer.id
-
-        except ElisaError as ex:
-            self.log.error(f"ELisA logbook: {str(ex)}")
-            self.log.error(answer)
-            raise ex
-        
-        self.log.info(f"ELisA logbook: Sent message (ID{self.current_id})")
-        
-        os.remove(sso['ssocookie'])
-
-
-    def message_on_start(self, message:str, apparatus:str, run_num:int, run_type:str):
-        self._start_new_message_thread()
-        self.current_run_num = run_num
-        self.current_run_type = run_type
-
-        text = f"<p>User {credentials.user} started run {self.current_run_num} of type {self.current_run_type} on {apparatus}</p>"
-        if message != "":
-            text += "\n<p>"+credentials.user+": "+message+"</p>"
-        text += "\n<p>log automatically generated by NanoRC.</p>"
-        
-        title = f"{credentials.user} started new run {self.current_run_num} ({self.current_run_type}) on {apparatus}"
-        self._send_message(subject=title, body=text, command='start')
-
-    def add_message(self, message:str, apparatus:str):
-        if message != "":
-            text = "<p>"+credentials.user+": "+message+"</p>"
-            self._send_message(subject="User comment", body=text, command='message')
+    def __init__(self, AID):
+        filepath = str(os.path.dirname(__file__)) + "/confdata/elisa_service.json"
```

Comment:
This should be using the module "resources" from `importlibs`. See example here: https://github.com/DUNE-DAQ/nanorc/blob/develop/src/nanorc/__main_np04__.py#L91